internal/core/services/room_manager/room_manager.go

```diff
@@ -115,7 +115,7 @@ func (m *RoomManager) DeleteRoom(ctx context.Context, gameRoom *game_room.GameRo
 func (m *RoomManager) UpdateRoom(ctx context.Context, gameRoom *game_room.GameRoom) error {
 	currentGameRoom, err := m.roomStorage.GetRoom(ctx, gameRoom.SchedulerID, gameRoom.ID)
 	if err != nil {
-		return fmt.Errorf("unable to fetch game room from storage: %w", err)
+		return err
```

Comment:
Any reason for dropping this error wrapping?

Reply:
Yes, we are handling this error at RoomsHandler to return with NotFound 404 when no game room is found.